### ISSUE TITLE
Go to feature crashes the app

### ISSUE THREAD

**Issue by llemeurfr** [edrlab/thorium-reader#1539]
I’m writing to report that in the Go to Page tool in the Navigation panel, when you press Enter on the keyboard, after entering the number in the "Enter a page number box", the program looks like crashing, turning to blank and you have to close it by pressing the "x" and re-open the file; you need to click the "Go to" rather than pressing "Enter" to navigate to the desired page.
 
I was wondering if that is a bug, or was it designed that way? Lots of times I just pressed Enter “automatically” after entering the number, only to have to close and re-open it.
 
Thanks again for the wonderful tool for reading EPUB!

**Comment by llemeurfr** [comment]
The issues is seen on v1.7.1; it is reproductible on MacOS.

**Comment by panaC** [comment]
![image](https://user-images.githubusercontent.com/3198865/132478503-f519c6ce-60e7-40eb-90b5-d3bf3f6fd8bd.png)

with childrens-literature from idpf

**Comment by panaC** [comment]
there are an issue on the button Go-to : 


shift + enter key press : 
![image](https://user-images.githubusercontent.com/3198865/132492707-dbe36a03-6ab3-4b54-8b0c-e8b8cadb44dc.png)
new thorium window appears

enter key press : 
the page is reloaded : works in debug mode but not in prod mode

shift + click on "Go To" or click without shift on "Go To" : 
redirection to the right page but the panel is always close

**Comment by danielweck** [comment]
Good catch! Fixed via PR https://github.com/edrlab/thorium-reader/pull/1540
